examples/dht_sensor/dht_sensor.c

```diff
@@ -0,0 +1,39 @@
+/* 
+ *
+ * This sample code is in the public domain.
+ */
+#include "espressif/esp_common.h"
+#include "esp/uart.h"
+#include "FreeRTOS.h"
+#include "task.h"
+#include "dht.h"
+#include "esp8266.h"
+
+/* An example using the ubiquitous DHT** humidity sensors
+ * to print out a new temperature and humidity measurement
+ * every three seconds.
+ */
+int const gpio4 = 4;
+
+void dhtMeasurementTask(void *pvParameters)
+{
+  int8_t temperature = 0;
+  int8_t humidity = 0;
+
+  gpio_set_pullup(gpio4, false, false);

```

Comment:
I'd be curious to know whether the internal pullups are sufficient for the DHT sensor (at least when connecting it over short wires directly to the pin, etc).. I suspect they are, in which case it might be more convenient to have the example default to having the internal pullup enabled instead..  Alternately, it might be good to put a comment in here pointing out that this assumes you have an external pullup already attached (and if you don't you'll want to change that setting)..  Not a big deal either way, though..